generators/generator-base.js

```diff
@@ -41,7 +41,7 @@ util.inherits(Generator, yeoman.Base);
  */
 Generator.prototype.addElementToMenu = function (routerName, glyphiconName, enableTranslation) {
     try {
-        var fullPath = CLIENT_MAIN_SRC_DIR + 'app/layouts/navbar/navbar.html';
+        var fullPath = CLIENT_MAIN_SRC_DIR + 'app/layouts/navbar/navbar.component.html';
```

Comment:
@deepu105 But just to be sure. Will Angular 1 will be supported in JHipster 4 ? I thought it would disappear (or will it stay but in BETA ?)